sourcemash/frontend/static/scss/sourcemash.scss

```diff
@@ -366,6 +366,10 @@ a.button-collapse.top-nav {
     .activator {
         cursor: pointer;
     }
+    img[width*="1px"] {
```

Comment:
I don't think thats necessary

Alex Gerstein
gerstein.alex@gmail.com




On Sun, May 17, 2015 at 4:28 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/static/scss/sourcemash.scss:
> @@ -366,6 +366,10 @@ a.button-collapse.top-nav {
>      .activator {
>          cursor: pointer;
>      }
> +    img[width*="1px"] {



Should you add this for height=1px too? 



—
Reply to this email directly or view it on GitHub.